### DISCUSSION TITLE
image sequence export

### DISCUSSION THREAD

**Discussion by jclevet** [mifi/lossless-cut#1634] [Ideas]
I love this software. The ability to export in image sequence would be a useful option.
JPG, or better EXR (DWA16float)...
thank you for your great work

**Comment by mifi** [comment]
Hi. Not sure what that means exactly? There is already a function to export frames as images (jpg/png)

**Comment by jclevet** [comment]
Sorry, I see the possibility of exporting a single image, but I did not find image sequence instead of a video format in the export parameters. Did I miss this option?

**Reply by mifi** [reply]
In the right segment list panel, right click on a segment, "Extract frames as images"

**Comment by jclevet** [comment]
Thank you ! This software is amazing !!!
